src/content/reference/app-defaulting-validation/index.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Defaulting and Validation of App CRs
+description: How defaulting and validation of app CRs is implemented by app-admission-controller
+weight: 110
+type: page
+last_review_date: 2020-12-16
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-batman
+user_questions:
+  - Is App CR defaulting and validation logic enabled for my cluster? 
+  - How can I use App CR defaulting logic when installing Managed Apps?
+  - What fields are validated in App CRs when installing or updating Managed Apps?
```

Comment:
> I would really appreciate if search questions be added to the front matter. This will help in our overall effort in making docs more searchable and accessible.

@Oshratn Yes good point. I've added these questions.